server/metadata.go

```diff
@@ -713,7 +718,10 @@ func (m *metadataAPI) ResumePartition(streamName string, id int32, recovered boo
 	}
 
 	// Resume the partition by replacing it.
-	partition, err := m.newPartition(partition.Partition, recovered)
+	// nil given as configuration means we do not set
+	// custom stream's configuration. The default broker configuration
+	// will be used.
```

Comment:
Why do we do this on partition resume?

Reply:
as `(* server) newParition()` signature changes to


```golang
func (s *Server) newPartition(protoPartition *proto.Partition, recovered bool, protoStreamsConfig *proto.CustomStreamConfig) (*partition, error)
```
so I have to pass `nil` here